app/webpacker/packs/application.js

```diff
@@ -1,11 +1,23 @@
 import "../styles/application.scss";
+import "accessible-autocomplete/dist/accessible-autocomplete.min.css";
 import { initAll } from "govuk-frontend";
 import initLocationsMap from "scripts/locations-map";
 import backLink from "scripts/back-link";
+import initAutocomplete from "scripts/autocomplete";
 
 initAll();
 
 window.initLocationsMap = initLocationsMap;
 
 const $backLink = document.querySelector('[data-module="back-link"]');
 new backLink($backLink).init();
+
+try {
+  const $autocomplete = document.getElementById("provider-autocomplete");
+  const $input = document.getElementById("query");
+  if ($autocomplete) {
```

Comment:
Seeing as initAutoComplete needs `$input`, I think should also check that it exists before trying to call initAutoComplete. 

I do agree with @defong, I think this block should be part of initAutoComplete to keep `application.js` clean and simple to read.

Reply:
👍

Reply:
👍